### ISSUE TITLE
Problem with arrows on colony production screen

### ISSUE THREAD

**Issue by ExultantMonkey** [BotF2/BotF2#116]
Reported by Bashir https://star-trek-supremacy.proboards.com/post/7711/thread

In colony management both my son and I found that the rounded arrows on the right don't pull unemployed population onto the respective production topic (Automated farm, Replication Plant, Plasma reactor, University, Data Bank). On the left edge only the top three decrease the population employed. The bottom two end caps / arrows on the left do not work to decrease population

**Comment by KennethGF** [comment]
Hope to look at this tomorrow.

**Comment by KennethGF** [comment]
got a null ref from UniverseObjects.cs EffectsBinding return value _effectBindings.Value line 436

**Comment by KennethGF** [comment]
Getting the impression of a failure pattern in the hover over step of buttons. Appears that when one button to increment is inactive the decrement is not. Clicking around with other buttons and the decrement increment pattern is reversed when you return to it.

**Comment by KennethGF** [comment]
Reset() is a function in  \BotF2\SupremacyUI\SystemProductionPanel.cs , or something like it to update, refresh, that might need to be added in some part of the code.

**Comment by KennethGF** [comment]
\SupremacyClient\OptionsPages\AllOptionsPage.xaml(47):   Command="{x:Static s:BlockBar.DecrementCommand}"

\themes\Borg\Shared.xaml(1440):   Command="{x:Static s:UnitActivationBar.DecrementValueCommand}"

SupremacyUI\BlockBar.cs(32):      DecrementCommand = new RoutedCommand("Decrement", typeof(BlockBar));

SupremacyUI\UnitActivationBar.cs(576):            if (_decrementButton != null)

**Comment by KennethGF** [comment]
Think the button is in UnitActivationBar, see the increment and decrement fields

**Comment by KennethGF** [comment]
Found that jumping to ShipBuilding tab and back to Colony Handling activates the increment right button. Also jumping to a new window, like Diplomacy, activates the button. Trying to use the SendKeys command to turn it on but hitting some conflicts and issues. 

                        SendKeys.SendWait("{F5}");
                        SendKeys.SendWait("{F2}");

**Comment by bote-reginald** [comment]
yeah... decreasing pop on Facilities (Industry, Energy ...) sometimes delivers strange results
